vision.md

```diff
@@ -0,0 +1,407 @@
+# What is our vision?
+
+The aim of creating thin-edge.io is to provide a IoT edge device framework for
+IoT project teams which makes it easy to enable resource constrained devices for
+IoT. Unlike other solutions, we are not just another single-purpose agent but a
+flexible platform with re-usable components without any vendor lock-in, focused
+on adressing the needs of both IT and OT users.
+
+## Our motivation
+
+We believe that IoT (edge/device management, middleware, data analytics) are the
+driving forces of the fourth industrial revolution: companies are forced to
+leverage IoT solutions to stay competitive. In the past a large amount of
+companies have suffered from failed IoT projects and initiatives, where a lot of
+time and money was spent on device enablement: connectivity, security, device
+management, etc...
+
+Thats why we are creating thin-edge.io. It provides an easy and flexible way for
+IoT project teams and device manufactures to make their devices IoT-ready
+without having to re-invent the wheel with costly and time-consuming,
```

Comment:
I am not sure I understand this sentence. Might be worth considering to split it into multiple sentences.

Proposal:

> Thats why we are creating thin-edge.io. It provides an easy and flexible way for IoT project teams and device manufactures to make their devices IoT-ready. This saves them time as they do not have to re-invent the wheel with costly and time-consuming, potentially embedded, development projects. Also the modular architecture does not force IoT project teams to start from scratch but thin-edge.io can be enabled for existing projects or applications.